src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -366,26 +366,53 @@ function showNewlyAddedResponseCommentEditForm(addedIndex) {
     $("#responseCommentEditForm-" + addedIndex).show();
 }
 
-function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
+function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, clickedElement) {
     $(".tooltip").hide();
-    var panelBody = $(sender).parent().find('div[class^="panel-body"]');
+    var $clickedElement = $(clickedElement);
+    var panelBody = $clickedElement.parent().find('div[class^="panel-body"]');
     var fsNameForUrl = encodeURIComponent(fsName);
     var url = "/page/instructorFeedbackResponseCommentsLoad?user=" + user + "&courseid=" + courseId + "&fsname=" + fsNameForUrl + "&fsindex=" + fsIndx;
-    $(sender).find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
-    panelBody.load(url, function( response, status, xhr ) {
-        if (status == "success") {
-            panelBody.removeClass('hidden');
-            updateBadgeForPendingComments(panelBody.children(":first").text());
-            panelBody.children(":first").remove();
-            registerResponseCommentsEvent();
-            registerCheckboxEventForVisibilityOptions();
-            enableHoverToDisplayEditOptions();
-      } else {
-          panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
-          panelBody.removeClass('hidden');
-      }
-      $(sender).find('div[class^="placeholder-img-loading"]').html("");
-    });
+    
+    if ($clickedElement.hasClass("loaded")) {
+        $clickedElement.siblings(".collapse").clearQueue().collapse("toggle");
+        toggleChevron(clickedElement);
```

Comment:
May be you can do an early return here. It will reduce nesting in the code below.